aws/src/main/java/org/frankframework/jms/AmazonSqsFactory.java

```diff
@@ -24,23 +24,24 @@
 import javax.naming.Reference;
 import javax.naming.spi.ObjectFactory;
 
+import jakarta.jms.Connection;
+import jakarta.jms.ConnectionFactory;
+import jakarta.jms.JMSException;
+
 import com.amazon.sqs.javamessaging.AmazonSQSMessagingClientWrapper;
 import com.amazon.sqs.javamessaging.ProviderConfiguration;
 import com.amazon.sqs.javamessaging.SQSConnection;
 import com.amazon.sqs.javamessaging.SQSConnectionFactory;
```

Comment:
Bij mij komt hier nog een newline tussen, dus:
```
import com.amazon.sqs.javamessaging.SQSConnectionFactory;

import lombok.Getter;
```

Verder heb ik geen changes tov jouw pr!

Reply:
Zit er in de intelllij config nog iets dat je na een com.*** import een newline moet stoppen?

Reply:
@nielsm5 , ja klopt:
```
        <package name="org" withSubpackages="false" static="false" />
        <emptyLine />
        <package name="com" withSubpackages="true" static="false" />
        <emptyLine />
```